### PR TITLE
Add healing boost to organs while mob is sleeping

### PR DESCRIPTION
**Pull request by timothymtorres**
## About The Pull Request
This adds the healing bonuses from sleeping effects (mood, pillow, sleeping in darkness, etc.) and applies it to organs natural healing regeneration.  The regen rate is very low, so I added a heavy x5 multiplication rate to all the sleep bonuses for organs specifically.

If an organ is dying or robotic then the effects don't apply.

## Why It's Good For The Game
More benefits to sleeping after injuries.  This encourages patients to rest properly after revival or major surgery.

## Changelog
:cl:
add: Add healing boost to organs while mob is sleeping (does not apply to robotic or dying organs)
/:cl:

### CODE REVIEW

**Comment by SkeletalElite** [comment]
Me on my way to survive being a chain smoker by taking a nap every once in a while

**Comment by SomeoneYouProbablyKnow** [comment]
> Me on my way to survive being a chain smoker by taking a nap every once in a while

Accurate

File: code/datums/status_effects/debuffs/debuffs.dm
Comment (ChungusGamer666) [inline code/datums/status_effects/debuffs/debuffs.dm:208]: currently, organs are not stored inside bodyparts unless they are external organs.
this will only work after #76381 is merged, and even then it should be target_bodypart.organs

Comment (timothymtorres) [reply code/datums/status_effects/debuffs/debuffs.dm:208]: Good to know.  I will wait until your PR gets merged before this continues.

File: code/datums/status_effects/debuffs/debuffs.dm
Comment (MrMelbert) [inline code/datums/status_effects/debuffs/debuffs.dm:207]: Why iterate over all bodyparts anyways, why not just iterate over `owner.organs`

Comment (ChungusGamer666) [reply code/datums/status_effects/debuffs/debuffs.dm:207]: Good question

Comment (timothymtorres) [reply code/datums/status_effects/debuffs/debuffs.dm]: Applied with e5c590f6d66c1f2d0f7d46a11c3b2f9b8c6bc1b2

File: code/datums/status_effects/debuffs/debuffs.dm
Comment (Time-Green) [inline code/datums/status_effects/debuffs/debuffs.dm:2]: Please autodoc any defines you add (or move) :)

**Review by Time-Green — CHANGES_REQUESTED** [review]

File: code/datums/status_effects/debuffs/debuffs.dm
Comment (timothymtorres) [reply code/datums/status_effects/debuffs/debuffs.dm:4]: Applied with https://github.com/tgstation/tgstation/pull/76679/commits/5c63188a4beb6d12f01bc2919f5a998b7ca4a221

**Pull request merged**
